### DISCUSSION TITLE
multiple example notebooks failing on 'get_image_uri'

### DISCUSSION THREAD

**Discussion by aaronmarkham** [aws/sagemaker-python-sdk#2368] [Help]
**Describe the bug**
ImportError: cannot import name 'get_image_uri'

**To reproduce**
from sagemaker import get_image_uri

**Expected behavior**
no error

**Comment by chuyang-deng** [comment]
Hi @aaronmarkham,

We just released Python SDK v2: https://pypi.org/project/sagemaker/

``get_image_uri`` has been deprecated and we recommend using ``image_uris.retrieve()``: https://github.com/aws/sagemaker-python-sdk/blob/master/src/sagemaker/image_uris.py

If you still want to use the old interface please pin sagemaker version < 2

**Comment by dz902** [comment]
SageMaker Studio seems to have Python SDK built-in, how do we determine which version it is on? Will this new version get an automatic roll out on Studio notebooks?
